### PR TITLE
feat: my kiva header

### PR DESCRIPTION
**Pull request by christian14b**
Screenshot 2024-09-17 at 3 43 27 PM

### CODE REVIEW

**Comment by dyersituations** [comment]
@christian14b could you add a mobile screenshot as well?

File: src/pages/Portfolio/MyKiva/MyKivaPage.vue [hunk 1]
```diff
@@ -1,33 +1,146 @@
...
 		);
+
+		this.$kvTrackEvent('portofolio', 'view', 'new-my-kiva');
+	},
+	methods: {
+		adjustAvatarPosition() {
+			const { landscape } = this.$refs;
+			if (landscape) {
+				const containerHeight = landscape.clientHeight;
+				this.avatarStyle.top = `${containerHeight - 40}px`;
```
Comment (dyersituations) [inline src/pages/Portfolio/MyKiva/MyKivaPage.vue:108]: Why is this needed for the avatar? Could this not be done with CSS, moving it up from the bottom?

File: src/pages/Portfolio/MyKiva/MyKivaPage.vue [hunk 2]
```diff
@@ -1,33 +1,146 @@
...
+		},
 	},
 };
 </script>
+
+<style lang="postcss" scoped>
+
+.landscape {
+	background-image: url('~@/assets/images/my-kiva/header-mobile.svg');
+	background-size: cover;
```
Comment (dyersituations) [inline src/pages/Portfolio/MyKiva/MyKivaPage.vue:127]: `tw-bg-cover`

File: src/pages/Portfolio/MyKiva/MyKivaPage.vue [hunk 3]
```diff
@@ -1,33 +1,146 @@
...
+	background-repeat: no-repeat;
```
Comment (dyersituations) [inline src/pages/Portfolio/MyKiva/MyKivaPage.vue:128]: `tw-bg-no-repeat`

**Comment by dyersituations** [comment]
It looks like two of the elements need to be better center aligned:
image

**Comment by roger-in-kiva** [comment]
Looks like the avatar border is not fully rounded when having an image and doesn't fit it:

Screenshot 2024-09-18 at 5 35 22 p m

**Comment by roger-in-kiva** [comment]
Looks like the container in the mocks is smaller than our current:

Screenshot 2024-09-18 at 5 58 50 p m

Screenshot 2024-09-18 at 5 59 02 p m

**Comment by dyersituations** [comment]
@christian14b please post screenshots of desktop and mobile of the latest changes.

File: src/components/MyKiva/MyKivaHero.vue [hunk 4]
```diff
@@ -0,0 +1,55 @@
+<template>
+	<div ref="landscape" class="tw-w-full landscape tw-relative">
```
Comment (roger-in-kiva) [inline src/components/MyKiva/MyKivaHero.vue:2]: Do we need this ref?

File: src/components/MyKiva/MyKivaHero.vue [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+		<kv-page-container class="!tw-max-w-4xl">
```
Comment (emuvente) [inline src/components/MyKiva/MyKivaHero.vue:3]: It looks like the max-width from the design is 1072px (67rem). The design team wants to try out a new page width with this project, so let's use some custom css to apply that specific max-width. That's also going to be the max width for all the sections on the page, so it would make sense to make a reusable component to hold the page container, grid, and the custom max-width css.

**Review by roger-in-kiva — APPROVED** [review]

**Review by emuvente — APPROVED** [review]
Looks great!
